packages/web3-eth-abi/package.json

```
@@ -0,0 +1,46 @@
+{
+	"name": "web3-eth-abi",
+	"version": "4.0.0-alpha.0",
+	"description": "Web3 module encode and decode EVM in/output.",
+	"main": "dist/index.js",
+	"repository": "https://github.com/ethereum/web3.js/tree/4.x/packages/web3-eth-abi",
+	"author": "ChainSafe Systems",
+	"license": "LGPL-3.0",
+	"files": [
+		"dist/**/*"
+	],
+	"scripts": {
+		"clean": "rimraf dist",
+		"prebuild": "rimraf dist",
+		"build": "tsc --build",
+		"build:check": "node -e \"require('./dist')\"",
+		"lint": "eslint --ext .js,.ts .",
+		"lint:fix": "eslint --fix --ext .js,.ts .",
+		"format": "prettier --write '**/*'",
+		"test": "jest --config=./test/unit/jest.config.js",
+		"test:coverage": "jest --config=./test/unit/jest.config.js --coverage=true --coverage-reporters=text",
+		"test:ci": "jest --coverage=true --coverage-reporters=json --verbose",
+		"test:watch": "npm test -- --watch",
+		"test:unit": "jest --config=./test/unit/jest.config.js",
+		"test:integration": "jest --config=./test/integration/jest.config.js"
+	},
+	"dependencies": {
+		"web3-utils": "4.0.0-alpha.0",
+		"web3-common": "1.0.0-alpha.0",
+		"@ethersproject/abi": "5.0.7"
```

```suggestion
		"@ethersproject/abi": "^5.0.7"
```

Was it an explicit decision to hardcode the version @nazarhussain?

---

Updated.